SU-SWS/react_paragraphs
# D8CORE-1419 Updated dependencies & add a row when dragging

# READY FOR REVIEW

# Summary
- Updated underlying depdendencies
- Added a row to the bottom of the list if the last row is full. this reduces one step for the user

# Need Review By (Date)
- no rush

# Urgency
- low

# Steps to Test
1. Checkout this branch
1. clear browser caches
1. edit/create a basic page
1. drag an item into the open row
1. without clicking the button to add another row, start to drag another item
1. see that a new row is created when you are dragging the item.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
